### PR TITLE
Release/r86

### PR DESCRIPTION
**Pull request by alexanderdean**
Hey @chuwy - this is the branch for the next release but it's in a bit of a mess (mostly self-inflicted by me):

* Somehow I can't find this commit in this branch, #e678d299fc7402a1a77d31bd5d94d8597c816653
* Somehow I can't find this commit in this branch, #c3bf6f1441bbf99d372eba7984917dc5748589e9
* This issue #2983 should also be reverting this commit, #22c60598b6d28ffb2fadaab1eef4760d3557101f

I suspect I did some kind of force pulling or somesuch - apologies. Hope you have a clean local copy.

Could you please get this prepped for release as a priority Monday? You will also need to cherry-pick over all of the data modeling tickets, which are moving up into this release:

* https://github.com/snowplow/snowplow/issues/2956
* https://github.com/snowplow/snowplow/issues/2954
* https://github.com/snowplow/snowplow/issues/2929
* https://github.com/snowplow/snowplow/issues/2925
* https://github.com/snowplow/snowplow/issues/2836
* https://github.com/snowplow/snowplow/issues/2626

Also do please add the CHANGELOG in a final "Prepared for release" commit.

And please delete branches as/when they don't contain anything useful...

Thanks! Please let me know when this release branch is ready to go...

### CODE REVIEW

**Comment by chuwy** [comment]
@alexanderdean it seems you pushed branch `feature/dedupe` which was removed github. Actual ready-for-release branch is `release/dedupe`.

**Comment by chuwy** [comment]
I cherry-picked data-modeling commits. All except #2929, because there's no commit yet.

/cc @bogaert

**Comment by chuwy** [comment]
@alexanderdean I need to force-push this branch now with ready `release/dedupe`. Did you change/add anything except #2963 and #2985?

**Comment by alexanderdean** [comment]
Thanks for sorting all this out @chuwy - my commits are #2963, #2985 **and** #2986.

**Comment by chuwy** [comment]
Ok, cool. Force-pushing!

**Comment by alexanderdean** [comment]
Hey @chuwy - can you liaise with @dmpacheco to get his avatar on #9c53ec3?

**Comment by chuwy** [comment]
@alexanderdean done!

**Comment by chuwy** [comment]
Travis failed because OpenWeatherMap didn't respond at time. Hope this is just short outage.

**Comment by chuwy** [comment]
Rebased with missed data-modelling commits.

/cc @bogaert

**Comment by chuwy** [comment]
Hey @alexanderdean! Can I rebase it one more time? I need to delete [`event-recovery-ci.bash`](https://github.com/snowplow/snowplow/blob/3271d073345747611e2fded20dcc12f6595c8e3c/event-recovery-ci.bash) as part of #2983.

**Comment by alexanderdean** [comment]
Sure - go for it!

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,28 @@
+Release 86 Petra (2016-12-20)
+---------------------------
```
Comment (alexanderdean) [inline CHANGELOG]: Missing two hyphens here @chuwy

**Pull request merged**